packages/core/src/JWEditor.ts

```diff
@@ -34,8 +36,12 @@ type CommandParams<T extends JWPlugin, K extends string> = K extends Commands<T>
     : never;
 
 export interface JWEditorConfig {
+    defaults?: {
+        Container?: new () => VNode & { atomic: false };
+        Atomic?: new () => VNode & { atomic: true };
+        Separator?: new () => VNode;
```

Comment:
Why isn't it `new AtomicNode()` etc.?

Reply:
Because people can do a node that is atomic without using our AtomicNode class.

Reply:
Why?